ui/server.tsx

```diff
@@ -7,10 +7,13 @@ import { App } from 'ui/App'
 import { Config, ConfigProvider } from 'ui/components/ConfigProvider'
 import { Document } from 'ui/Document'
 import { initApollo } from 'ui/lib/initApollo'
-
 export default async function(req: Request, res: Response, config: Config) {
   let html = ''
-
+  const stuf = '../public/assets.json'
+  const scripts = Object.entries(((await import(stuf)).default) as { client: string }).map(([, src]) => ({
```

Comment:
Move the `import` statement out of this line to clean things up. It's really hard to read as-is.